### ISSUE TITLE
Certain code causes vscode to freeze

### ISSUE THREAD

**Issue by derekwisong** [jshinonome/vscode-q#70]
If I enter the following code into an empty .q file and save it - vscode freezes.

```
{f[{key x} x;1;x]}
```

VSCode version: 1.67.0
vscode-q version: 2.5.5   (also tried the previously working for me, on older vscode, version 2.3.6)

This seemed to start happening after my organization recently updated vscode versions to 1.67.0

In case it helps...
```
{f[{};1;x]}     - does not freeze
{f[key x;1;x]}  - does not freeze
{f[{key x};1;x]} - does not freeze
{f[{key x} x;1;x]}  - CAUSES VSCODE TO FREEZE
```

This is also reproducible in Remote SSH to Linux and locally on Windows.

A variety of scripts are now causing vscode to freeze when I open them. I presume there are more situations than the above causing lockups because so many of my scripts now cause vscode to freeze when I open them.

**Comment by derekwisong** [comment]
In case it helps...

```
{f[{};1;x]}     - does not freeze
{f[key x;1;x]}  - does not freeze
{f[{key x};1;x]} - does not freeze
{f[{key x} x;1;x]}  - CAUSES VSCODE TO FREEZE
```

This is also reproducible in Remote SSH to linux and Windows.

**Comment by jshinonome** [comment]
Hi, the issue seems to be with Bracket pair colorization, I will raise a ticket with vscode.
you can add the following to *Open Setting (JSON)* as a temporary fix.
```
    "[q]": {
        "editor.language.colorizedBracketPairs": []
    },
}
```

**Comment by derekwisong** [comment]
Thanks Jo. I am using the workaround successfully.

**Comment by jshinonome** [comment]
You are welcome. Hopefully vscode team would solve this issue soon.

**Comment by jshinonome** [comment]
@derekwisong the issue should be fixed now(v2.5.8), let me know if you still see the issue.
